### DISCUSSION TITLE
how to achieve cloudwatch - log tail (live)

### DISCUSSION THREAD

**Discussion by MushiTheMoshi** [awslabs/aws-sdk-rust#1058] [Q&A]
Hi there,

Awesome work done here, thanks.

Looking to read from a log stream and start a live session as per this [guide](https://docs.aws.amazon.com/code-library/latest/ug/cloudwatch-logs_example_cloudwatch-logs_StartLiveTail_section.html)

I think is not documented in the sdk, can you please point me in the right direction?

Regards,
Julio

**Comment by rcoh** [comment]
Seems like a good candidate for an example.
